lib/OcrProcessors/OcrMyPdfBasedProcessor.php

```diff
@@ -94,9 +100,13 @@ public function ocrFile(File $file, WorkflowSettings $settings, GlobalSettings $
 			throw new OcrNotPossibleException('OCRmyPDF did not produce any output');
 		}
 
+		if (!$this->recognizedTextFile || !$recognizedText = file_get_contents($this->recognizedTextFile)) {
+			$recognizedText = '';
```

Comment:
A log `warning` message like 

```suggestion
                        $recognizedText = '';
                        $this->logger->warning('Temporary sidecar file at \'{path}\' could not be created or was empty', ['path' => $this->recognizedTextFile]);
```

would make sense to me here for debugging purposes. What do you think?

- [ ] Log message
- [ ] Testcase should be added checking if a warning is logged if the file cannot be created or was empty

Reply:
absolutely!